src/gen/js/genTypes.ts

```diff
@@ -268,6 +268,7 @@ function renderTypeDoc(name: string, def: any): string[] {
   if (propLines.length) lines.push(`${DOC}`)
   join(lines, propLines)
   lines.push(' */')
+  lines.push(`const ${name}=null`)
```

Comment:
Can you clarify this one? It's not need to add code in my testing. Also setting to null causes issues for people not allowing null's in their code base.

Reply:
I'm not sure if it is removed by code minification, but potentially it can hurt "anyone" because you're potentially shipping more bytes through the wire.